README.md

```diff
@@ -587,6 +587,94 @@ Once you opt-in to use additional OCI hooks, any new orchestrated containers wil
+* `settings.oci-defaults.capabilities.lease`: process capability `CAP_LEASE`
+* `settings.oci-defaults.capabilities.linux-immutable`: process capability `CAP_LINUX_IMMUTABLE`
+* `settings.oci-defaults.capabilities.mac-admin`: process capability `CAP_MAC_ADMIN`
+* `settings.oci-defaults.capabilities.mac-override`: process capability `CAP_MAC_OVERRIDE`
+* `settings.oci-defaults.capabilities.mknod`: process capability `CAP_MKNOD`
+* `settings.oci-defaults.capabilities.net-admin`: process capability `CAP_NET_ADMIN`
+* `settings.oci-defaults.capabilities.net-bind-service`: process capability `CAP_NET_BIND_SERVICE`
+* `settings.oci-defaults.capabilities.net-broadcast`: process capability `CAP_NET_BROADCAST`
+* `settings.oci-defaults.capabilities.net-raw`: process capability `CAP_NET_RAW`
+* `settings.oci-defaults.capabilities.perfmon`: process capability `CAP_PERFMON`
+* `settings.oci-defaults.capabilities.setgid`: process capability `CAP_SETGID`
+* `settings.oci-defaults.capabilities.setfcap`: process capability `CAP_SETFCAP`
+* `settings.oci-defaults.capabilities.setpcap`: process capability `CAP_SETPCAP`
+* `settings.oci-defaults.capabilities.setuid`: process capability `CAP_SETUID`
+* `settings.oci-defaults.capabilities.sys-admin`: process capability `CAP_SYS_ADMIN`
+* `settings.oci-defaults.capabilities.sys-boot`: process capability `CAP_SYS_BOOT`
+* `settings.oci-defaults.capabilities.sys-chroot`: process capability `CAP_SYS_CHROOT`
+* `settings.oci-defaults.capabilities.sys-module`: process capability `CAP_SYS_MODULE`
+* `settings.oci-defaults.capabilities.sys-nice`: process capability `CAP_SYS_NICE`
+* `settings.oci-defaults.capabilities.sys-pacct`: process capability `CAP_SYS_PACCT`
+* `settings.oci-defaults.capabilities.sys-ptrace`: process capability `CAP_SYS_PTRACE`
+* `settings.oci-defaults.capabilities.sys-rawio`: process capability `CAP_SYS_RAWIO`
+* `settings.oci-defaults.capabilities.sys-resource`: process capability `CAP_SYS_RESOURCE`
+* `settings.oci-defaults.capabilities.sys-time`: process capability `CAP_SYS_TIME`
+* `settings.oci-defaults.capabilities.sys-tty-config`: process capability `CAP_SYS_TTY_CONFIG`
+* `settings.oci-defaults.capabilities.syslog`: process capability `CAP_SYSLOG`
+* `settings.oci-defaults.capabilities.wake-alarm`: process capability `CAP_WAKE_ALARM`
+
+##### OCI Defaults: Resource Limits
+
+Each of the `resource-limits` settings below contain two numeric fields: `.hard-limit` and `.soft-limit`, which are **32-bit unsigned integers**.
+
+Please see the `getrlimit` linux manpage for meanings of `.hard-limit` and `.soft-limit`.
+
+The **default** resource limits that are set in Bottlerocket are:
+
+setting key | setting value
+----- | -----
+`settings.oci-defaults.resource-limits.max-open-files.hard-limit` | 1048576
+`settings.oci-defaults.resource-limits.max-open-files.soft-limit` | 65536
+
+* `settings.oci-defaults.resource-limits.max-open-files`: resource limit `RLIMIT_NOFILE`
```

Comment:
I meant to mirror the capabilities section's "defaults then full list" pattern -- I forgot to add "The full list of resource limits that can be configured in Bottlerocket are"... before the listing.

Would adding that line before the (1-item) list of resource limits work? Or would it still be better to put the list of resource limits before the Bottlerocket defaults list? (My thinking that is if, in the future, ALL resource limits become configurable, but not all have default values, it would be better for both capabilities and resource-limits to have the shorter defaults section first, then the full list after that).

What do you think?

Reply:
What you have now reads better. Thanks